src/main/java/com/iota/iri/controllers/TransactionViewModel.java

```diff
@@ -104,7 +104,7 @@ public TransactionViewModel(final int[] trits, Hash hash) {
     public TransactionViewModel(final byte[] bytes, Hash hash) throws RuntimeException {
         transaction = new Transaction();
         transaction.bytes = new byte[SIZE];
-        System.arraycopy(bytes, 0, transaction.bytes, 0, SIZE);
+        System.arraycopy(bytes, 0, transaction.bytes, 0, bytes.length);
```

Comment:
Hey,
When I first saw this change before the merge I thought it was unnecessary but harmless so I didn't say anything.
This change however broke IRI.

For the future, refactoring changes should be submitted in a seperate PR.
Thanks!!